frontend/src/components/App/settings/hook.tsx

```diff
@@ -0,0 +1,13 @@
+import { useEffect, useState } from 'react';
+import { useTypedSelector } from '../../../redux/reducers/reducers';
+
+export const useSettings = function (settingName: string) {
+  const storeSettingNameVal = useTypedSelector(state => state.ui.settings[settingName]);
```

Comment:
The settingName should be optional. So if none is provided, you should get all settings.